lib/Twig/Extension/AdminExtension.php

```diff
@@ -49,6 +57,18 @@ public function getFilters()
         ];
     }
 
+    /**
+     * @return ?string
+     */
+    public function getAdminLanguage()
```

Comment:
We should use typehints instead of php doc
```suggestion
    public function getAdminLanguage(): ?string
```